examples/3d/scattering3d_qm7.py

```diff
@@ -208,61 +206,56 @@ def compute_qm7_solid_harmonic_scattering_coefficients(
         full_density_batch = generate_weighted_sum_of_gaussians(
                 grid, pos_batch, full_batch, sigma, cuda=cuda)
         full_order_0 = compute_integrals(full_density_batch, integral_powers)
-        full_order_1, full_order_2 = scattering(
+        full_scattering = scattering(
                 full_density_batch, order_2=True, method='integral',
                 integral_powers=integral_powers)
 
         val_density_batch = generate_weighted_sum_of_gaussians(
                 grid, pos_batch, val_batch, sigma, cuda=cuda)
         val_order_0 = compute_integrals(val_density_batch, integral_powers)
-        val_order_1, val_order_2 = scattering(
+        val_scattering= scattering(
                 val_density_batch, order_2=True, method='integral',
                 integral_powers=integral_powers)
 
         core_density_batch = full_density_batch - val_density_batch
         core_order_0 = compute_integrals(core_density_batch, integral_powers)
-        core_order_1, core_order_2 = scattering(
+        core_scattering = scattering(
                 core_density_batch, order_2=True, method='integral',
                 integral_powers=integral_powers)
 
 
         order_0.append(
             torch.stack([full_order_0, val_order_0, core_order_0], dim=-1))
-        order_1.append(
-            torch.stack([full_order_1, val_order_1, core_order_1], dim=-1))
-        order_2.append(
-            torch.stack([full_order_2, val_order_2, core_order_2], dim=-1))
+        orders_1_and_2.append(
+            torch.stack(
+                [full_scattering, val_scattering, core_scattering], dim=-1))
 
     order_0 = torch.cat(order_0, dim=0)
-    order_1 = torch.cat(order_1, dim=0)
-    order_2 = torch.cat(order_2, dim=0)
+    orders_1_and_2 = torch.cat(orders_1_and_2, dim=0)
 
-    return order_0, order_1, order_2
+    return order_0, orders_1_and_2
 
 M, N, O, J, L = 192, 128, 96, 2, 3
```

Comment:
I see the `shape` is used within the function above for creating the scattering object. So this example runs, right? (I'm fine keeping MNO here until going over this to create a narrative)